packages/imperative/__tests__/config/cmd/init/init.handler.test.ts

```diff
@@ -23,6 +23,7 @@ import * as lodash from "lodash";
 import * as fs from "fs";
 import * as os from "os";
 import { CredentialManagerFactory } from "../../../../../security";
+import { param } from "jquery";
```

Comment:
Could you please remove this unused import?

Reply:
Removed it